website/src/components/Home/AboutSection.js

```diff
@@ -6,18 +6,19 @@ import "../../base.css";
 export default function Header() {
   return (
     <div className='about'>
-      <h1 className='title'> About</h1>
-      <p className='main-text'>
-        Lorem ipsum dolor sit amet, consectetur adipiscing elit, sed do eiusmod
-        tempor incididunt ut labore et dolore magna aliqua. Ut enim ad minim
-        veniam, quis nostrud exercitation ullamco laboris nisi ut.
+    <h1 className='title'> About</h1>
+    <p className='main-text'>
+      Blueprint strives to make technology accessible and useful for those who assist communities and promote public welfare. 
+    </p>
+    <p className='extra-text'>
+      <p>
+        At Blueprint, we are dedicared and driven to develop pro bono apps for nonprofits and promotes tech for social good, free of charge. 
       </p>
-      <p className='extra-text'>
-        Lorem ipsum dolor sit amet, consectetur adipiscing elit, sed do eiusmod
-        tempor incididunt ut labore et dolore magna aliqua. Ut enim ad minim
-        veniam, quis nostrud exercitation ullamco laboris nisi ut.
+      <p>
+        Blueprint is a national organization founded in 2012 at the UC Berkeley. UC Irvine is the fourth established chapter, founded in 2019 after University of Waterloo and UCLA.
```

Comment:
Change "the UC Berkeley" to just "UC Berkeley"

Reply:
international* b/c of waterloo!!